web_development_101/javascript_basics/project_calculator.md

```diff
@@ -639,4 +639,5 @@ Submit a solution with a pull request to this [file](https://github.com/TheOdinP
 - [Dennis Kenneth’s Solution](https://github.com/denniskenneth/Calculator) - [View in Browser](https://denniskenneth.github.io/Calculator/)
 - [AnsellMaximilian's Solution](https://github.com/AnsellMaximilian/calculator) - [View in Browser](https://ansellmaximilian.github.io/calculator/)
 - [Wil Goolsby's Solution](https://github.com/wgoolsby/calculator) - [View in Browser](https://wilgoolsby.com/calculator/)
+-[Jonathan Fallas's Solution](https://github.com/Elfjhon/calculator)- [View in Browser](https://elfjhon.github.io/calculator/)
```

Comment:
Requesting Changes

Reply:
change done!!! thanks and sorry about it